Contest_Package.md

```diff
@@ -63,6 +63,37 @@ and `contest/<filename>` respectively, and files referenced in
 Note that the API specification requires that filenames are unique within
 endpoint objects, so this is always possible.
 
+#### Default filenames
+
+In some situations you may want file references to be found on disk
+without directly specifying them in a json file, for instance to keep
+json files simple or when providing organization logos or team photos
+manually or from different systems. Files that use the default
+filename pattern must be loaded in addition to any file references
+found in the json files.
+
+The default filename pattern is
+`<endpoint>/<id>/<property>(.variant).<extension>`, where:
+
+- `<id>` is the ID of the endpoint object the reference is in.
+- `<property>` is the property of the object, e.g. `logo`.
+- `(.<variant>)` is an optional variant. Must be used when there are
```

Comment:
I think this will depend on the choice made in https://github.com/icpc/ccs-specs/issues/202: for example if that would become an array of strings, then probably this should become `(.<variant>)*`. Bottom-line is that a filename should be uniquely mappable to a file reference object, I think.

Reply:
I agree - but this PR should be simple / match what we've been doing, and leave #202 to extend it.